tasks/mrrc-collect-params/mrrc-collect-params.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: mrrc-collect-params
+spec:
+  params:
+    - name: dataJsonPath
+      description: path to data json file
+  results:
+    - name: mrrcParamFilePath
+      description: path of the env file for following tasks
+    - name: cosignPubKeyFile
+      description: the public key file name for cosign verification
+    - name: cosignPubKeySecret
+      description: the secret name for cosign verify public key
+    - name: charonAWSSecret
+      description: the secret name for charon aws credential file
+  steps:
+    - name: mrrc-collect-params
+      image: quay.io/konflux-ci/yq:latest
+      script: |
+        set -eux
+
+        cd $(workspaces.data.path)
+        mrrcEnvFile="./mrrc.env"
+        if [ -f $mrrcEnvFile ]; then
```

Comment:
Why is this needed? I would suggest to remove it.

Reply:
Fixed in new commit